### PR TITLE
add logit API

### PR DESCRIPTION
**Pull request by wangzhen38**
### PR types
 New features 

### PR changes
 APIs

### Describe
add logit API，you can use this function as follows:
```
// logit(x) = ln(1/(1-x))
import paddle
x = paddle.to_tensor([0.23, 0.45])
out = paddle.logit(x)
```
截屏2021-12-08 下午7 45 26

截屏2021-12-08 下午7 45 43

### CODE REVIEW

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -2614,6 +2614,60 @@ def atan2(x, y, name=None):
                 type='atan2', inputs=inputs, outputs={'Out': out})
         return out
 
+def logit(x, eps=0.0, name=None):
```
Comment (jeff41404) [inline python/paddle/tensor/math.py]: default of eps should be None?

**Comment by jeff41404** [comment]
this is elementwise operator, better to add in activation_op?

File: python/paddle/fluid/tests/unittests/test_logit_op.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+# limitations under the License.
+
+import unittest
+import numpy as np
+from op_test import OpTest
+import paddle
+np.random.seed(10)
+
+
+def logit(x, eps):
```
Comment (jeff41404) [inline python/paddle/fluid/tests/unittests/test_logit_op.py:22]: shall we add test case when eps is None?

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (wangzhen38) [reply python/paddle/tensor/math.py]: DONE

**Comment by wangzhen38** [comment]
> this is elementwise operator, better to add in activation_op?

DONE

File: python/paddle/fluid/tests/unittests/test_logit_op.py (hunk 2 above)
Comment (wangzhen38) [reply python/paddle/fluid/tests/unittests/test_logit_op.py:22]: DONE

**Review by jeff41404 — APPROVED** [review]
lgtm

**Review by Superjomn — APPROVED** [review]
LGTM

**Review by dingjiaweiww — APPROVED** [review]
LGTM
